native/acter/src/api/room.rs

```diff
@@ -644,6 +688,10 @@ impl Room {
                     }
                     SyncOrStrippedState::Sync(ev) => ev.event_id().to_owned(),
                 };
+                let permitted = room.can_user_redact_own(&my_id).await?;
```

Comment:
needs the same redact check as above.

Reply:
Fixed by https://github.com/acterglobal/a3/pull/1671/commits/3ec8b833f13bc2974c6a896e7223b64068b081b3